package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "emuflight-configurator",
   "description": "Crossplatform configuration tool for Emuflight flight control system.",
-  "version": "0.2.24",
+  "version": "0.2.25",
```

Comment:
please leave the version bumps out of PRs. this will be done on merges to master and/or releasing from there